tests/data-type.image.spec.js

```diff
@@ -0,0 +1,47 @@
+const expect = require('chai').expect;
+const Image = require('../src/data_type/image');
+const ImageJpeg = require('../src/data_type/image.jpeg');
+const ImagePng = require('../src/data_type/image.png');
+
+const imageContent = {
+  toBase64() {
+    return 'test_image_data';
+  },
+};
+
+const imageTypes = [
+  {
+    imageObj: new ImageJpeg(imageContent),
+    mimeType: 'image/jpeg',
+  },
+  {
+    imageObj: new ImagePng(imageContent),
+    mimeType: 'image/png',
+  },
+];
+
+imageTypes.forEach((type) => {
+  describe(`${type.imageObj.constructor.name}`, () => {
+    context('#getContent()', () => {
+      it('it should return original image content', () => {
```

Comment:
I think these tests shouldn't have the second `it`, so for this one it would just read as `it('should return original image content`

Reply:
I've now corrected this

Reply:
I was talking about the other `it`s in this class too, it's in a few other places 😃

Reply:
Now corrected the other occurrences 👍